XSConsoleRemoteTest.py

```diff
@@ -13,27 +13,27 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-import os, socket, xmlrpclib
+import os, socket, xmlrpc.client
 
 from XSConsoleBases import *
 from XSConsoleImporter import *
 from XSConsoleLang import *
 from XSConsoleLog import *
 from XSConsoleLayout import *
 
-import SocketServer
-import SimpleXMLRPCServer
+import socketserver
+import xmlrpc.server
```

Comment:
I advocate for this simple solution to replace this entire commit with these two hunks.

The main reason is that the the python-future module became unmaintained and projects are already forced to drop their use of python-future because of this when they want to support Python 3.12. One Example:

https://github.com/gkovacs/lz-string-python/pull/6/files

A fixup for this particular issue of future using `import imp` which has been dropped has been submitted 3 weeks ago, but the maintainers of future have gone missing in January, which is now 9 months without any commit or merge. The deprecation warning from the imp module has been spread wide from the stable release of 3.7 to 3.11, with already 3.10 specifically warning that  imp will not be killed with 3.12, which it did:

```py
# python3.10 -Wdefault -c 'from future import standard_library;standard_library.install_aliases()'
/usr/lib/python3/dist-packages/future/standard_library/__init__.py:65: DeprecationWarning: the imp module is deprecated in favour of importlib and slated for removal in Python 3.12; see the module's documentation for alternative uses
  import imp
# python3.12 -c 'from future import standard_library;standard_library.install_aliases()'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/future/standard_library/__init__.py", line 65, in <module>
    import imp
ModuleNotFoundError: No module named 'imp'
```
Therefore, just use the simple try: import method and no change to the actual code.

That's fully in line with Andrew's demand for Python2 compat and it avoids having dependency on no longer maintained packages (there is also six.moves, but that also is a not needed dependency which can become unmaintained). Besides, when pylint would used (may be good when developing new code), two pylint warnings would have to be disabled, because installing the aliases would have to be put on top, violating the import order:
```py
# pylint: disable=wrong-import-position,wrong-import-order
from future import standard_library
standard_library.install_aliases()
```